src/routers/alpha-router/alpha-router.ts

```diff
@@ -1471,7 +1471,60 @@ export class AlphaRouter
     const V3poolsRaw = V3poolAccessor.getAllPools();
     const V2poolsRaw = V2poolAccessor.getAllPools();
 
-    const poolsRaw = [...V3poolsRaw, ...V2poolsRaw];
+    /// not just [Protocol.MIXED]
+    let chosenV2Pools: Pair[] = [];
+    if (
+      !routingConfig.forceMixedRoutes &&
```

Comment:
Why do we not want this logic when mixed routes are forced?

Reply:
I think because if we are forcing mixedRoutes, we want to return ANY mixed route available, not necessarily only one that matches this heuristic criteria. I think the main use case for this that I envisioned was for the `protocols: [V3, V2, MIXED]` one and the `[]` case.